ts/types/MobileControlsComponent.d.ts

```diff
@@ -0,0 +1,19 @@
+interface ControlAbility {
```

Comment:
how can these all be optional? do we sometimes have
```ts
controls: {
    abilities: {
        control: {}
    }
}
```
?

Reply:
also w/h?

Reply:
at the time when I added that it was the most accurate representation of objects used as control abilities.